build.sh

```diff
@@ -13,6 +13,8 @@
 set -o pipefail
 set -e
 
+source ./scripts/bandwidth_throttling.sh
```

Comment:
the `bandwidth_throttling.sh` script doesn't appear to be used anywhere in `build.sh`, so why are we sourcing it here?